spiffworkflow-backend/src/spiffworkflow_backend/routes/script_assist_controller.py

```diff
@@ -0,0 +1,54 @@
+from os import environ
+from flask import make_response
+from flask import request
+from flask.wrappers import Response
+from openai import OpenAI
+
+# TODO: We could just test for the existence of the API key, if it's there, it's enabled.
+# Leaving them separate now for clarity.
+# Note there is an async version in the openai lib if that's preferable.
+def enabled() -> Response:
+    response = environ.get("SPIFFWORKFLOW_SCRIPT_ASSIST_ENABLED");
+    result = response in ["True", "true", "1"]
+    return make_response({"ok": result}, 200)
+
+def process_message() -> Response:
+    openai_api_key = environ.get("SECRET_KEY_OPENAI_API")
+    if not openai_api_key:
+        return make_response({"ok": "OpenAI API key not set"}, 200)
```

Comment:
My feeling is no, because the quest did indeed complete properly. By keeping the HTTP status accurate, you know the call completed successfully, so back-end validation likely created an error message for the user. 400 Would indicate a broken request, which would first get you started on checking all your client side (which is fine).